populus/compilation/backends/solc_standard_json.py

```diff
@@ -114,10 +114,15 @@ def get_compiled_contracts(self, source_file_paths, import_remappings):
                 'remappings': import_remappings
             }
         }
-        std_input['settings'].update(self.compiler_settings)
 
+        std_input['settings'].update(self.compiler_settings.get("solidity_input_description", {}))
```

Comment:
I think that calling this `"standard_input"` is a good name/key for this setting.